tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../../Resources/Page/OCPLogin/OCPLogin.resource
+Resource        ../../../Resources/OCP.resource
+Test Setup      Dashboard Test Setup
+Test Teardown   Dashboard Test Teardown
+
+*** Variables ***
+${pachyderm_container_name}     Pachyderm
+${pachyderm_appname}            pachyderm
+
+*** Test Cases ***
+Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
+    [Tags]  Tier-2
```

Comment:
Tier tags shouldn't have "-" character. IIt should be **Tier2** without space. Please refer style guide for reference https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.4a2fb94fltk2